eclipse-scout-core/src/testing/menu/MenuSpecHelper.ts

```diff
@@ -16,11 +16,10 @@ export class MenuSpecHelper {
     this.session = session;
   }
 
-  createModel(text?: string, iconId?: string, menuTypes?: string[]): FullModelOf<Menu> {
+  createModel(text?: string, menuTypes?: string[]): FullModelOf<Menu> {
```

Comment:
Adjust commit comment and add that MenuSpecHelper was simplified by removing iconId